src/utils/caseStateFilters.ts

```diff
@@ -6,5 +6,6 @@ export const caseStateLabels: KeyValuePair<string, string> = {
 }
 
 const caseStateFilters: string[] = ["Unresolved and resolved", "Resolved"]
+export const myCaseStateFilters = "View cases allocated to me"
```

Comment:
I think we could delete this const and just cast the query param into a boolean

Reply:
All done!